js/src/dropdown.js

```diff
@@ -1,3 +1,5 @@
+/* global Popper */
```

Comment:
I see, but I don't understand how does it handle the `import` and `export` logic...
It first transpiles each module to Require.js and then it concats them together right? But where?